documentation/ntfs/ntfs_0.6.md

```diff
@@ -0,0 +1,705 @@
+
+(1) Les valeurs possibles sont :
+
+* 1 - Le service est ajouté pour cette date
+* 2 - Le service est supprimé pour cette date
+
+### comments.txt (optionnel)
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+comment_id | chaine | Requis | Identifiant du commentaire
+comment_type | chaine | Optionnel | (1)
+comment_name | chaine | Requis | Texte du commentaire
+comment_url | chaine | Optionnel | URL associé à la note et permettant d'avoir plus d'info, comme par exemple un lien vers la page de description du service de TAD.
+
+(1) Catégorie de commentaire afin de pouvoir les différentier à l'affichage. Les valeurs possibles sont :
+
+* Information (ou non renseigné) : indique une note d'information générale
+* OnDemandTransport : indique qu'il s'agit d'une note d'information sur le Transport à la demande. Ce type de notes doit préciser de manière succinte les conditions et le numéro de téléphone de réservation
+
+### comment_links.txt (optionnel)
+Ce fichier fait le lien entre un objet du référentiel (ligne, arrêt, horaire, etc.) et un commentaire afin de permettre d'associer plusieurs notes à un objet. et plusieurs objets à une note.
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+object_id | chaine | Requis | Identifiant de l'objet associé à la note
+object_type | chaine | Requis | Type de l'objet associé au commentaire. Les valeurs possibles sont stop_area, stop_point, line, route, trip, stop_time ou line_group.
+comment_id | chaine | Requis | Identifiant du commentaire (lien vers le fichier comments.txt)
+
+### commercial_modes.txt (requis)
+Ce fichier décrit les modes commerciaux, c'est à dire un libellé particulier de mode de transport. Par exemple, BusWay est un nom particulier de BHNS à Nantes.
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+commercial_mode_id | chaine | Requis | Identifiant du mode commercial
+commercial_mode_name | chaine | Requis | Nom du mode commercial
+
+### companies.txt (requis)
+Ce fichier décrit l'opérateur de transport exploitant tout ou partie d'un des réseaux contenus dans les données.
+
+Colonne | Type | Contrainte | Commentaire
+--- | --- | --- | ---
+company_id | chaine | Requis | Identifiant de la compagnie
+company_name | chaine | Requis | Nom de la compagnie
+company_address_name | chaine | Optionnel | Nom de la voie de l'adresse
+company_address_number | chaine | Optionnel | Numéro d'adresse
+company_address_type | chaine | Optionnel | Type de voie (rue, avenue, etc.)
+company_url | chaine | Optionnel | Url du site institutionnel de la société. A ne pas confondre avec le lien vers le site du réseau.
+company_mail | chaine | Optionnel | Adresse mail de contact de la société
+company_phone | chaine | Optionnel | Numéro de téléphone de contact
+company_fax | chaine | Optionnel | Numéro de fax
+
+### contributors.txt (réservé)
+Ce fichier permet d'identifier la ou les sources fournissant les données du présent jeu de données.
```

Comment:
All references to contributor_id is modified to "optional" except for the Frame reference